runtime/compiler/runtime/Listener.cpp

```diff
@@ -348,9 +348,38 @@ static int32_t J9THREAD_PROC listenerThreadProc(void * entryarg)
       {
       OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
       char timestamp[32];
+      char zoneName[32];
+      int32_t zoneSecondsEast = 0;
+      TR_VerboseLog::CriticalSection vlogLock;
 
       omrstr_ftime_ex(timestamp, sizeof(timestamp), "%b %d %H:%M:%S %Y", j9time_current_time_millis(), OMRSTR_FTIME_FLAG_LOCAL);
-      TR_VerboseLog::writeLineLocked(TR_Vlog_INFO, "StartTime: %s", timestamp);
+      TR_VerboseLog::writeLine(TR_Vlog_INFO, "StartTime: %s", timestamp);
+
+      TR_VerboseLog::write(TR_Vlog_INFO, "TimeZone: ");
+      if (-1 == omrstr_current_time_zone(&zoneSecondsEast, zoneName, sizeof(zoneName)))
+         TR_VerboseLog::write("(unavailable");
+      else
+         {
+         /* Writing UTC[+/-]hr:min (zoneName) to the log */
+         uint32_t offset = ((uint32_t)((zoneSecondsEast > 0) ? zoneSecondsEast : -zoneSecondsEast)) / 60;
+         uint32_t hours = offset / 60;
+         uint32_t minutes = offset % 60;
+         TR_VerboseLog::write("UTC");
+
+         if (0 != zoneSecondsEast)
+            {
+            const char *format = (zoneSecondsEast > 0) ? "+%d" : "-%d";
+            uint32_t offset = ((uint32_t)((zoneSecondsEast > 0) ? zoneSecondsEast : -zoneSecondsEast)) / 60;
+            uint32_t hours = offset / 60;
+            uint32_t minutes = offset % 60;
```

Comment:
The type of these three variables should be `int32_t`; the coercion to unsigned is not necessary and does not match the formats being used.

Reply:
I took that from the existing code in `javadump.cpp`

https://github.com/eclipse-openj9/openj9/blob/master/runtime/rasdump/javadump.cpp#L704-L706

Would you like me to change them there too?

Reply:
No, the difference there is that the values will be passed to `writeInteger()` which has an unsigned parameter.